app/controllers/practice_partners_controller.rb

```diff
@@ -11,6 +11,16 @@ def index
   # GET /practice_partners/1.json
   def show
     @facilities_data = VaFacility.cached_va_facilities.order_by_station_name
+    @partner_practices = helpers.is_user_a_guest? ? @practice_partner.practices.published_enabled_approved.public_facing.sort_by_retired.sort_a_to_z : @practice_partner.practices.published_enabled_approved.sort_by_retired.sort_a_to_z
```

Comment:
Not sure if it would make a huge difference, but should we consider using `searchable_practices`/`searchable_public_practices` here?

Reply:
ooo good point! thanks Josh!